static/js/components/VegaPlot.jsx

```diff
@@ -59,7 +59,7 @@ const spec = (url) => ({
           type: "nominal"
         },
         tooltip: [
-          { field: "magAndErr", title: "mag" },
+          { field: "magAndErr", title: "mag", type: "nominal" },
```

Comment:
I see that the type is actually required, was it the reason why it was throwing errors?